source/_integrations/unifiprotect.markdown

```diff
@@ -54,9 +54,34 @@ UCKP with Firmware v1.x **do NOT run UniFi OS**, you must upgrade to firmware v2
 
 ### Software Support
 
-The absolute **minimal** software version is `1.20.0` for UniFi Protect. If you have an older version, you will get errors trying to set up the integration. However, the general advice is the latest 2 minor versions of UniFi Protect and hardware supported by those are supported. Since UniFi Protect has its own release cycle, you should only upgrade UniFi Protect _after_ the next Home Assistant release comes out to ensure the new version is fully supported.
+The absolute **minimal** software version is `1.20.0` for UniFi Protect. If you have an older version, you will get errors trying to set up the integration. However, the general advice is the latest 2 minor versions of UniFi Protect and hardware supported by those are supported.
 
-Example: as of `2022.2.0` of Home Assistant, UniFi Protect `1.21.0` is the newest UniFi Protect version. So the recommended versions of UniFi Protect to run for a `2022.2.0` version of Home Assistant are `1.20.0`, `1.20.1`, `1.20.2`, `1.20.3`, `1.21.0`.
+#### About UniFi Early Access
+
+Since UniFi Protect has its own release cycle, you should only upgrade UniFi Protect _after_ the next Home Assistant release comes out to ensure the new version is fully supported. Most importantly, that means **do not use Early Access versions of UniFi Protect if you want your UniFi Protect integration to be stable**. Early Access version can and will break the UniFi Protect Home Assistant in unexpected ways. If you really want to use Early Access version of UniFi Protect, you can disable automatic updates and wait for the next bugfix version of UniFi Protect to come out.
```

Comment:
```suggestion
Since UniFi Protect has its own release cycle, you should only upgrade UniFi Protect _after_ the next Home Assistant release to ensure the new version is fully supported. Most importantly, that means **do not use Early Access versions of UniFi Protect if you want your UniFi Protect integration to be stable**. The Early Access version can and will break the UniFi Protect Home Assistant unexpectedly. If you desire to use the Early Access version of UniFi Protect, you can disable automatic updates and wait for the next bugfix version of UniFi Protect to come out.
```